cloud/azure/bin/setup-saml-keystore

```diff
@@ -39,8 +38,8 @@ if [[ ! "${resource_group}" ]] \
   exit 1
 fi
 
-if key_vault::has_secret "${vault_name}" "${KEYSTORE_PASS_SECRET_NAME}" &&
-  storage::has_blob "${storage_account_name}" \
+if key_vault::has_secret "${vault_name}" "${KEYSTORE_PASS_SECRET_NAME}" \
+                                                                        && storage::has_blob "${storage_account_name}" \
```

Comment:
question (blocking): is there a setting we can change that won't result in such deep indentation for continuations?